project/Version.scala

```diff
@@ -6,13 +6,12 @@ object Version {
   val baseModelVersion = "0.1"
 
   def getVersionString = {
-    def getLastModelCommitFromGit = { s"""git log -n 1 --pretty=format:%h""" !! }
```

Comment:
need hash even for publishing rawls model. this handles the hash without needing to install git on the docker image (which, fun fact, does not have apt, yum, dpkg, or even sudo)

 ```
 14:42:55 [info] resolving key references (11031 settings) ...
 14:42:55 java.io.IOException: Cannot run program "git": error=2, No such file or directory
 14:42:55 	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1128)

```
from https://fc-jenkins.dsp-techops.broadinstitute.org/job/rawls-build/4805/console

Reply:
Out of curiosity, why the change from `def` to `lazy val`?